.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Node Build
+ 
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+ 
+jobs:
+  build:
+    runs-on: ubuntu-latest
+ 
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Node
+        uses: actions/setup-node@v4
+ 
+      - name: install dependencies
+        run: npm install
+      - name: Run CI
+        run: npm ci
```

Comment:
is npm install required with npm ci?

Reply:
I thought it was required because that's how we did it last time but after some research I don't think it's necessary. I have removed it